LICENSE-3rdparty.csv

```diff
@@ -1,1408 +1,1408 @@
+core,github.com/gosnmp/gosnmp,BSD-2-Clause,The GoSNMP authors
+core,github.com/grpc-ecosystem/go-grpc-middleware,Apache-2.0,"Gengo, Inc"
+core,github.com/grpc-ecosystem/go-grpc-middleware/auth,Apache-2.0,"Gengo, Inc"
+core,github.com/grpc-ecosystem/go-grpc-middleware/util/metautils,Apache-2.0,"Gengo, Inc"
+core,github.com/grpc-ecosystem/go-grpc-prometheus,Apache-2.0,"Gengo, Inc"
+core,github.com/grpc-ecosystem/grpc-gateway/codegenerator,BSD-3-Clause,"Gengo, Inc"
+core,github.com/grpc-ecosystem/grpc-gateway/internal,BSD-3-Clause,"Gengo, Inc"
+core,github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway,BSD-3-Clause,"Gengo, Inc"
+core,github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway/descriptor,BSD-3-Clause,"Gengo, Inc"
+core,github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway/generator,BSD-3-Clause,"Gengo, Inc"
+core,github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway/gengateway,BSD-3-Clause,"Gengo, Inc"
+core,github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway/httprule,BSD-3-Clause,"Gengo, Inc"
+core,github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger,BSD-3-Clause,"Gengo, Inc"
+core,github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger/genswagger,BSD-3-Clause,"Gengo, Inc"
+core,github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger/options,BSD-3-Clause,"Gengo, Inc"
+core,github.com/grpc-ecosystem/grpc-gateway/runtime,BSD-3-Clause,"Gengo, Inc"
+core,github.com/grpc-ecosystem/grpc-gateway/utilities,BSD-3-Clause,"Gengo, Inc"
+core,github.com/h2non/filetype,MIT,Tomas Aparicio
+core,github.com/h2non/filetype/matchers,MIT,Tomas Aparicio
+core,github.com/h2non/filetype/matchers/isobmff,MIT,Tomas Aparicio
+core,github.com/h2non/filetype/types,MIT,Tomas Aparicio
+core,github.com/hashicorp/consul/api,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/errwrap,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/go-cleanhttp,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/go-hclog,MIT,HashiCorp
+core,github.com/hashicorp/go-immutable-radix,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/go-multierror,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/go-rootcerts,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/golang-lru,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/golang-lru/simplelru,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/hcl,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/hcl/hcl/ast,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/hcl/hcl/parser,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/hcl/hcl/printer,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/hcl/hcl/scanner,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/hcl/hcl/strconv,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/hcl/hcl/token,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/hcl/json/parser,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/hcl/json/scanner,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/hcl/json/token,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/serf/coordinate,MPL-2.0,UNKNOWN
```

Comment:
397 - 416.  HashiCorp Inc. 

based off notice in https://github.com/hashicorp/consul/blob/main/NOTICE.md 

assuming the rest are the same, though they all don't have notices.